containerm/util/containers_parse.go

```diff
@@ -83,25 +84,42 @@ func ParseMountPoints(mps []string) ([]types.MountPoint, error) {
 }
 
 // ParseMountPoint converts a single string representation of a container's mount to a structured MountPoint instance.
-// Format: source:destination[:propagation_mode].
+// Format: source:destination[:propagation_mode]:encoded_config_file_information.
 // If the propagation mode parameter is omitted, rprivate will be set by default.
 // Available propagation modes are: rprivate, private, rshared, shared, rslave, slave.
-func ParseMountPoint(mp string) (*types.MountPoint, error) {
+func ParseMountPoint(mp string) (*types.MountPoint, []byte, error) {
 	mount := strings.Split(strings.TrimSpace(mp), ":")
-	if len(mount) < 2 || len(mount) > 3 {
-		return nil, log.NewErrorf("Incorrect number of parameters of the mount point %s", mp)
+	if len(mount) < 2 || len(mount) > 4 {
+		return nil, []byte(""), log.NewErrorf("Incorrect number of parameters of the mount point %s", mp)
```

Comment:
```suggestion
		return nil, nil, log.NewErrorf("incorrect number of parameters of the mount point %s", mp)
```

Errors should not start with a capital letter. Ref: https://go.dev/wiki/CodeReviewComments#error-strings